### ISSUE TITLE
OK button not visible

### ISSUE THREAD

**Issue by dalb8** [nathanpc/Build.prop-Editor#4]
When editing a value the button in the top corner must be pressed to submit it. The problem is , it almost invisible.

**Comment by nathanpc** [comment]
What's the resolution of your device?

Anyway, I'll implement one of those "Unsaved Changes" dialogs, since I
already have the code from CherryNotes.

**Comment by dalb8** [comment]
800*480

**Comment by nathanpc** [comment]
That resolution is ok for the weight of the icon. Anyway I'll implement the
dialog.

**Comment by dalb8** [comment]
Its the colour that's the problem, I should have said that. It should be white, not dark grey.

**Comment by nathanpc** [comment]
Oh, you're using the new dark theme?

**Comment by dalb8** [comment]
There is only dark theme on 2.0.1, built from source.

**Comment by nathanpc** [comment]
Yeah, that was in the latest commit. We should've implemented a switch
button.
